public/js/index.js

```diff
@@ -3,3 +3,82 @@
  *= require jquery/dist/jquery.js
  *= require bootstrap/dist/js/bootstrap.js
  */
+
+ var lock = new Auth0Lock('pZJp3rVJ8wRvVLrX30HEOOHZ3HSEASec', 'hexlet-interviews.eu.auth0.com');
```

Comment:
Maybe better create credentials.js and import it here and use secret and callback properties? Also credentials.js could obtain this params from ENV, so no one will get them)